static-router.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const { join } = require('path');
+
+const router = express.Router();
+
+const staticDir = join(__dirname, 'static');
+
+// qr-code generator
+const qrCodeDir = join(staticDir, 'qrcode');
+router.use(express.static(qrCodeDir));
```

Comment:
you can set the `index` property of `express.static`, this should accomplish the same thing. Although `['index.html']` is its default value, so it should work even without that - am I missing something?